src/appleseed/renderer/kernel/lighting/ilightingengine.cpp

```diff
@@ -59,6 +59,14 @@ void ILightingEngineFactory::add_common_params_metadata(
                 .insert("label", "Light Samples")
                 .insert("help", "Number of samples used to estimate direct lighting"));
 
+		metadata.dictionaries().insert(
+			"dl_low_light_threshold",
```

Comment:
There are still tabs here.

Make sure your editor is configured to use 4 spaces for indenting.